### DISCUSSION TITLE
Carrot should instead be a banana

### DISCUSSION THREAD

**Discussion by karimk123** [monkeytypegame/monkeytype#1172] [Ideas] [answered]
this is monkeytype and monkeys eat and love bananas so carrot should be a banana instead.

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
carrot is easter

monkey celebrate easter

but banana i like

**Reply by Miodec** [reply]
Bananas are also curved, so it will be difficult to make it not cover up some of the letters - but sure, might add it

**Reply by karimk123** [reply]
You could maybe have it optional or make it less curved so it doesnt cover. I think it would be a nice touch
